shakenfist/external_api/app.py

```diff
@@ -139,8 +139,8 @@ def wrapper(*args, **kwargs):
                 ) as e:
             return error(401, str(e))
 
-        except Exception:
-            return error(500, 'server error')
+        except Exception as e:
+            return error(500, 'server error: %s' % e.__repr__())
```

Comment:
Normally you'd just use e here:

```'server error: %s' % e```

Even if the repr is needed you'd normally write it like this:

```'server error: %s' % repr(e)```